nautobot/core/templates/utilities/templatetags/saved_view_modal.html

```diff
@@ -7,11 +7,13 @@
                 <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
                 <h4 class="modal-title">New Saved View</h4>
             </div>
-            <form action="{% url 'user:savedview_add' %}?view={{view}}&{{params}}" method="post">
+            <form action="{% url 'user:savedview_add' %}" method="post">
                 <div class="modal-body">
                     {% csrf_token %}
                     <div class="form-group">
                         {% render_field form.name %}
+                        <input type="hidden" id="view" name="view" value="{{view}}">
+                        <input type="hidden" id="params" name="params" value="{{params}}">
```

Comment:
Did you test whether we need to `urlencode` `params` here if it contains special characters?

Reply:
Special characters are already encoded in `params` as you can see in the screenshot, e.g. `#` to `%23`. We actually have to decode it in the backend with `unquote` from `urllib.parse`
<img width="795" alt="Screenshot 2024-05-10 at 7 59 49 PM" src="https://github.com/nautobot/nautobot/assets/46973263/4e692a0f-7f79-4758-859b-e5aed24f5d2d">